api/config/base/apiconfig/korifi_api_config.yaml

```diff
@@ -7,7 +7,7 @@ defaultLifecycleConfig:
   stack: cflinuxfs3
   stagingMemoryMB: 1024
   stagingDiskMB: 1024
-packageRegistryBase: gcr.io/cf-relint-greengrass/cf-k8s-controllers/kpack/beta
+packageRegistryBase: gcr.io/cf-relint-greengrass/korifi-controllers/kpack/beta
```

Comment:
Do we need some extra action to make sure `korifi-controllers` exists inside `gcr.io/cf-relint-greengrass/`?